buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidTarget.groovy

```diff
@@ -254,9 +251,8 @@ abstract class AndroidTarget extends JavaLibTarget {
         return (builder.bind {
             mkp.yield manifestXml
         } as String)
-                .replaceAll('\\{http://schemas.android.com/apk/res/android\\}', 'android:')
-                .replaceAll('xmlns:android="http://schemas.android.com/apk/res/android"', '')
-                .replaceFirst('<manifest ', '<manifest xmlns:android="http://schemas.android.com/apk/res/android" ')
+            .replaceAll('xmlns:android="http://schemas.android.com/apk/res/android"', '')
```

Comment:
This is to handle manifests that don't declare android namespace. The build was previously falling when merging https://github.com/uber/okbuck/blob/master/libraries/emptyResourceLibrary/src/main/AndroidManifest.xml